x-pack/plugins/security_solution/public/detection_engine/rule_management/api/api.ts

```diff
@@ -678,26 +678,17 @@ export const performInstallSpecificRules = async (
     }),
   });
 
-export const performUpgradeAllRules = async (): Promise<PerformRuleUpgradeResponseBody> =>
-  KibanaServices.get().http.fetch(PERFORM_RULE_UPGRADE_URL, {
-    method: 'POST',
-    version: '1',
-    body: JSON.stringify({
-      mode: 'ALL_RULES',
-      pick_version: 'TARGET',
-    }),
-  });
-
```

Comment:
Deleting this since from now onwards the frontend client will never be just sending a request to update ALL rules, since some of them might have conflicts: we want to filter those out before making the request, which is done in this PR.

API users can still send this type request, and will get detailed errors if there are conflicts.